code/modules/clothing/under/jobs/civilian.dm

```diff
@@ -224,13 +224,15 @@
 
 /obj/item/clothing/under/rank/miner
 	desc = "It's a snappy jumpsuit with a sturdy set of overalls. It is very dirty."
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
 	name = "shaft miner's jumpsuit"
 	icon_state = "miner"
 	item_state = "miner"
 	item_color = "miner"
 
 /obj/item/clothing/under/rank/miner/lavaland
 	desc = "A green uniform for operating in hazardous environments."
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)

```

Comment:
Not required since /rank/miner/lavaland is a child of /rank/miner so it keeps the same armor as its parent

Reply:
That explains a lot, my bad